opera/rules.go

```diff
@@ -222,6 +225,7 @@ type Upgrades struct {
 	// -- Sonic Chain Hard Forks --
 	Sonic   bool // < launch version of the Sonic chain, introducing Cancun features
 	Allegro bool // < first hard fork of the Sonic chain, introducing Prague features
+	Brio    bool // < second hard fork of the Sonic chain, introducing Osaka features
```

Comment:
A bit surprised to find this in this PR, but I guess it is fine to introduce this with this PR. 

What should be added in a follow-up PR (see #243 as an example):
- in the rule validation, as a hard-fork, it should not be allowed to disable it one it is enabled
- coverage in the `marshal_test.go`
- coverage in the `rules_test.go`